src/Docfx.MarkdigEngine.Extensions/QuoteSectionNote/QuoteSectionNoteParser.cs

```diff
@@ -176,13 +176,29 @@ private bool TryParseFromLine(BlockProcessor processor, QuoteSectionNoteBlock bl
 
         if (infoString.StartsWith("[!Video", StringComparison.OrdinalIgnoreCase))
         {
-            string link = infoString.Substring(7, infoString.Length - 8);
-            if (link.StartsWith(" http://") || link.StartsWith(" https://"))
+            string data = infoString.Substring(7, infoString.Length - 8).Trim();
+            if (data.StartsWith("http://") || data.StartsWith("https://"))
             {
+                string link = data;
+                string title = "";
+                if (data.IndexOf(' ') != -1)
+                {
+                    link = data.Substring(0, data.IndexOf(' '));
+                    title = data.Trim().Substring(data.IndexOf(' ') + 1);
+                    if (title.StartsWith("title=") && title.Length > 6)
+                    {
+                        title = title.Substring(6);
+                    }
+                }
+
+                if (title == "") _context.LogWarning("inaccessible-video-section", "Video should have a valid url and title", block);
```

Comment:
Its possible this message might be misleading because this warning is just when the title is missing. Might be a better error to just say "Video title is required." or "Video title is missing. Please provide a video title."

Reply:
Yeah good call that is better :)